workflow/scripts/neighborhoods.py

```diff
@@ -457,21 +458,15 @@ def count_bam(
 
 def count_tagalign(tagalign, bed_file, output, genome_sizes):
     index_file = tagalign + ".tbi"
-    if os.path.exists(index_file):
-        command1 = ""
-    else:
-        command1 = "tabix -p bed {tagalign} | cut -f1-3".format(**locals())
+    if not os.path.exists(index_file):
+        cmd = f"tabix -p bed {tagalign} | cut -f1-3"
+        check_call(cmd, shell=True)
 
-    command2 = 'bedtools coverage -counts -b {tagalign} -a {bed_file} | awk \'{{print $1 "\\t" $2 "\\t" $3 "\\t" $NF}}\' '.format(
-        **locals()
-    )
-
-    p1 = Popen(command1, stdout=PIPE, shell=True)
+    cmd1 = f"bedtools coverage -counts -g {genome_sizes} -sorted -b {tagalign} -a {bed_file}"
```

Comment:
@engreitz thankfully pointed out that when we need to supply the genome file here. 

Doing a grep, I noticed there's a lot of `bedtools intersect` usage that we can potentially add the `-sorted` flag to improve memory efficiency. We'll have to remember to also add the `-g` flag there too